src/views/nodes-list.njk

```diff
@@ -2,13 +2,20 @@
 
 {% block content %}
 <main>
+  <header class="main-header">
+    <div class="main-title">
+      <h1>All the nodes</h1>
```

Comment:
"All the nodes" is a bit meme ;)

![](https://cdn.meme.am/cache/instances/folder750/500x/75958750.jpg)